packetbeat/Makefile

```diff
@@ -1,6 +1,6 @@
 
 BEATNAME?=packetbeat
-BEAT_DESCRIPTION?=A network analyzer tool that sends data to Elasticsearch.
+BEAT_DESCRIPTION?=Packetbeat: A network analyzer tool that sends data to Elasticsearch.
```

Comment:
Since it's not consistent with the other descriptions, how about "Packetbeat analyzes network traffic and sends the data to Elasticsearch."